src/widget/sidebar.h

```diff
@@ -2,6 +2,7 @@
 #define WIDGET_SIDEBAR_H
 
 #include "input/mouse.h"
+#include "widget/sidebar_filler.h"
```

Comment:
Why do we need this include?

Reply:
cause of the fun i had with constants; see prior comment